packages/klighd-core/src/proxy-view/proxy-view-cluster.tsx

```diff
@@ -0,0 +1,150 @@
+                                "ns": "http://www.w3.org/2000/svg",
+                                "style": {
+                                    "opacity": "1"
+                                },
+                                "attrs": {
+                                    "width":${squareSize},
+                                    "height":${squareSize},
+                                    "stroke": "black",
+                                    "fill": "rgb(220,220,220)"
+                                }
+                            },
+                            "children": []
+                        },
+                        {
+                            "sel": "g",
+                            "data": {
+                                "ns": "http://www.w3.org/2000/svg",
+                                "attrs": {
+                                    "id": "${id}4"
+                                }
+                            },
+                            "children": []
+                        }
+                    ]
+                },
+                ${"" /* Text containining number of proxies in cluster */}
+                {
+                    "sel": "g",
+                    "data": {
+                        "ns": "http://www.w3.org/2000/svg",
+                        "attrs": {
+                            "id": "keith-diagram_sprotty_$${id}2",
+                            "transform": "translate(${textSize * 0.5}, ${textSize})"
+                        },
+                        "class": {
+                            "selected": false
+                        }
+                    },
+                    "children": [
+                        {
+                            "sel": "g",
+                            "data": {
+                                "ns": "http://www.w3.org/2000/svg",
+                                "attrs": {
+                                    "id": "${id}5"
+                                }
+                            },
+                            "children": [
+                                {
+                                    "sel": "text",
+                                    "data": {
+                                        "ns": "http://www.w3.org/2000/svg",
+                                        "style": {
+                                            "dominant-baseline": "middle",
+                                            "font-family": "overpass, sans-serif",
+                                            "font-size": "${textSize}",
+                                            "opacity": 1
+                                        },
+                                        "attrs": {
+                                            "x": 0,
+                                            "xml:space": "preserve",
+                                            "y": 0,
+                                            "lengthAdjust": "spacingAndGlyphs"
```

Comment:
This attribute does nothing if you do not set a predefined text width. Can be removed.